### DISCUSSION TITLE
RAD causing no POST?

### DISCUSSION THREAD

**Discussion by Christophlex** [frntc/RAD#37] [Q&A]
Hello!  I purchased a RAD expansion unit (it's one of the suggested vendors - just their Ebay site), and when it's plugged in, with or without a Pi 3A+ or Zero 2 plugged in, there was no display whatsoever from the C64.  The monitor wouldn't detect a signal.

I found that there were 2 pairs of bridged pins on the LVC573 closest to the GPIO header.  I cleaned those up, and there's no longer continuity on those pairs.  Everything else looks good, and now the monitor at least detects a signal, but there's still no display, with or without a Pi plugged into the RAD.  The power from C64 header isn't jumpered (it's set for external power).

My breadbin 64 works great with all other cartridges and my Easy Flash 3.

My question is, should the RAD be causing no display on my 64, even when there is no Pi plugged in?  I figure the RAD is probably hosed, and I've messaged the seller, but I just wanted to know if that is normal behavior or not.  Thanks!

**Comment by frntc** [comment]
When you plug in the RAD without a RPi it means that RPi-side inputs to ICs (e.g. the level shifters, pull ups for disabling ICs) are floating. It might well be this pulls some signals on the C64 bus which prevents, e.g. the CPU, from running. I would not recommend doing that ;-)

**Comment by SensoryTrash** [comment]
try this 
[RAD_Firmware_v011Pi 2022.zip](https://github.com/frntc/RAD/files/14039277/RAD_Firmware_v011Pi.2022.zip)

**Comment by giobbino** [comment]
I had the same problem when I built mine first one; my fault, I didn't notice there are a couple of IC (iirc 573 and 30) that are opposite respect the silked label (there's a small dot). It caused a shortcut. Check it and see if all the ICs are soldered in the correct verse.  BTW at least one of the (30, iirc) became very hot (of course) and this gave me the clue.
